packages/node_modules/@node-red/editor-client/src/js/ui/palette-editor.js

```diff
@@ -625,12 +627,18 @@ RED.palette.editor = (function() {
             }
         })
 
+        // Add the update status to the status bar
+        addUpdateInfoToStatusBar();
+        // Load the catalogue and check for updates
+        getSettingsPane();
```

Comment:
Just spotted this - I assume this is done to trigger a load of the catalogues? This means the editor is now doing lots of additional work to generate the settings pane even though the user hasn't opened the dialog. We should only generate the settings pane if they are opening the Palette Manager.

We need to separate out loading the catalogues from generating the UI.

Reply:
> I assume this is done to trigger a load of the catalogues?

Yes

> We need to separate out loading the catalogues from generating the UI.

Ok, I will do that.